L/LibDeflate/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libdeflate"
```

Comment:
This name _must_ match exactly the name of the directory, case included

Reply:
Dear God I am not good at this. Thank you for your help. So it must be called `libdeflate-1.6` (name of the source directory), or `LibDeflate` (name of this JLL)?

Reply:
The name of the directory where the script is.  You can either rename the directory `libdeflate` or change back this variable to `LibDeflate`.  Usually we want to use the same name as the upstream package, which seems to be `libdeflate`?